src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -223,18 +223,23 @@ public void AllowReadStreamBuffering_SetTrueThenGet_ExpectTrue(Uri remoteServer)
             Assert.True(request.AllowReadStreamBuffering);
         }
 
-        [Theory, MemberData(nameof(EchoServers))]
-        public async Task ContentLength_Get_ExpectSameAsGetResponseStream(Uri remoteServer)
+        [Fact]
```

Comment:
This is causing us to lose testing of both http and https; now it's just validating http.  It'd be good to have both.

Reply:
Is there a recommended pattern with the loopbackserver to test both?

Reply:
> Is there a recommended pattern with the loopbackserver to test both?

You can, for example, pass in a bool with an InlineData(false) and InlineData(true), and then use that to initialize the LoopbackServer.Options passed in, e.g. `new LoopbackServer.Options { UseSsl = secure }`.

If that doesn't work for some reason, we could stick with just http.

Reply:
See example:
https://github.com/dotnet/corefx/blob/1064d0352c007fd2d8fb32d7f6e286fbb6d41264/src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs#L504-L514

Reply:
Thanks!